src/components/flat-table/flat-table.stories.mdx

```diff
@@ -181,8 +189,146 @@ import { FlatTable, FlatTableHead, FlatTableBody, FlatTableRow, FlatTableHeader,
   </Story>
 </Preview>
 
-## Props:
+### With transparent effect
+To create a transparent effect for the header, override the header background styling
+to match the same color as the background
 
+<Preview>
+  <Story name="with transparent effect" parameters={{ info: { disable: true }}} >
+    <div style={{background: '#FF785B', padding: '10px'}}>
+      <FlatTable styleOverride={{headerBackground: '#FF785B'}}>
```

Comment:
See above comment about `styleOverride`s

Reply:
fixed in https://github.com/Sage/carbon/pull/2866/commits/840d7cd4461c614101f121bf05a818cb440074d9